zetaclient/bitcoin_client.go

```diff
@@ -719,7 +735,55 @@ func (ob *BitcoinChainClient) observeOutTx() {
 	}
 }
 
-func (ob *BitcoinChainClient) BuildSubmittedTxMap() error {
+// Returns true if outTx passes basic checks
```

Comment:
```suggestion
// checkTssOutTxResult returns nil if outTx passes basic checks
```
I would suggest describing in the function docs what are the basic checks

Reply:
Sure. I added a bit more function doc.